test/lib/jdk/test/lib/util/ForceGC.java

```diff
@@ -32,21 +32,23 @@
  * Utility class to invoke System.gc()
  */
 public class ForceGC {
-    private final CountDownLatch cleanerInvoked = new CountDownLatch(1);
-    private final Cleaner cleaner = Cleaner.create();
+    private final static Cleaner cleaner = Cleaner.create();
+
+    private final CountDownLatch cleanerInvoked;
     private Object o;
 
     public ForceGC() {
         this.o = new Object();
-        cleaner.register(o, () -> cleanerInvoked.countDown());
+        this.cleanerInvoked = new CountDownLatch(1);
+        cleaner.register(o, cleanerInvoked::countDown);
     }
 
-    private void doit(int iter) {
+    private void doIt(int iter) {
         try {
-            for (int i = 0; i < 10; i++) {
+            for (int i = 0; i < 100; i++) {
                 System.gc();
-                System.out.println("doit() iter: " + iter + ", gc " + i);
-                if (cleanerInvoked.await(1L, TimeUnit.SECONDS)) {
+                System.out.println("doIt() iter: " + iter + ", gc " + i);
```

Comment:
Maybe the trace should be printed only if `(i % 10 == 0) && (iter % 100 == 0)` to avoid having too many traces in the log?

Reply:
This log was added to help debugging some past issues.  With this change, the volume of this trace would not be that helpful.   I think we can remove this log at this time.